conf/nginx/templates/nginx.conf.web.https.template

```diff
@@ -90,6 +90,18 @@ server
 
     location /services/files/
     {
+        set $public_link 0;
+        set $AUTH_COOKIE_MISSING 0;
+        if ($uri ~* "^/services/files/link/.*$")  {
```

Comment:
See above

Reply:
now both /service/files/link and /service/files/public are managed as URLs that do not require redirect